src/main/java/com/easypost/service/EasyPostClient.java

```diff
@@ -125,8 +166,17 @@ public String getApiKey() {
     }
 
     /**
-     * Get API base for this EasyPostClient object.
+     * Get API version for this EasyPostClient object.
      * 
+     * @return the API version for this EasyPostClient object.
+     */
+    public String getApiVersion() {
+        return apiVersion;
+    }
+
+    /**
+     * Get API base for this EasyPostClient object.
+     *
      * @return the API base for this EasyPostClient object.
      */
     public String getApiBase() {
```

Comment:
We don't need getters for the API base. Make the `apiBase` property public and final, so it can't be overridden by end-users (only set via the constructor)

Reply:
see above answer

Reply:
At least make the apiBase final

Reply:
I will make this tiny change in next PR so I don't dismiss the approvals